### PR TITLE
Update beta codes to greek chars

### PR DESCRIPTION
**Pull request by irina060981**
For the issue https://github.com/alpheios-project/alpheios-core/issues/104

Added feature - update beta codes in lookup to greek chars:

 - it is visible only for Greek
 - you should check update beta codes to turn on converting
 - on desktop you could see info by clicking small help icon

Desktop

image

image

image

image

Mobile

image

image

Added feature - update beta codes in lookup to greek chars:

 - it is visible only for Greek
 - you should check update beta codes to turn on converting
 - on desktop you could see info by clicking small help icon

Desktop

image

image

image

image

Mobile

image

image

image

### CODE REVIEW

**Comment by irina060981** [comment]
@balmas, as I started this PR yesterday - it is a descendant of that branch.
I have created a PR to the branch `i169-google-docs-mouse-over-bug` to make review simpler.

Of course I would merge it to master after `i169-google-docs-mouse-over-bug`.

File: packages/components/src/locales/en-us/messages.json [hunk 1]
```diff
@@ -686,5 +686,20 @@
     "message": "Show Language Selector",
     "description": "Tooltip for Show Languges Titles Block",
     "component": "Grammar"
+  },
+  "LOOKUP_USE_BETA_CODES_CHECK": {
+    "message": "update beta codes",
```
Comment (balmas) [inline packages/components/src/locales/en-us/messages.json:691]: please change the text to  "Enable input in betacode"

File: packages/components/src/locales/en-us/messages.json [hunk 2]
```diff
@@ -686,5 +686,20 @@
...
+    "description": "Update beta codes to greek chars",
+    "component": "Lookup"
+  },
+  "LOOKUP_USE_BETA_CODES_INFO_FIRST": {
+    "message": "Type in unicode polytonic Greek using beta code as indicated below:",
```
Comment (balmas) [inline packages/components/src/locales/en-us/messages.json:696]: Please change the text to "You can type Greek using the following betacode characters. They will be converted upon input to Unicode."

File: packages/components/src/locales/en-us/messages.json (hunk 1 above)
Comment (irina060981) [reply packages/components/src/locales/en-us/messages.json]: fixed
image

File: packages/components/src/locales/en-us/messages.json (hunk 2 above)
Comment (irina060981) [reply packages/components/src/locales/en-us/messages.json]: fixed

**Comment by irina060981** [comment]
Will close it and reopen for merging to master branch

**Pull request closed without merging**